src/index.html

```diff
@@ -5,8 +5,260 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
```

Comment:
тут используются размеры 400, 500, 700 - их все надо импортировать, а то проблемы с размерами на тестах вылезут
```suggestion
    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
```

Reply:
Супер професиональный ревю, делал ночю, сонный а ты все подметил. Спасибо)